### DISCUSSION TITLE
How much time does make launch take to finish?

### DISCUSSION THREAD

**Discussion by itserror404** [carla-simulator/carla#6621] [Q&A]
Hello, How much time does make launch take to finish?

Currently, it is stuck here and not sure if it's working or if there might be some issues.

<img width="1072" alt="Screenshot 2023-07-08 at 8 39 24 PM" src="https://github.com/carla-simulator/carla/assets/68902873/0417332e-b2f8-4228-bb99-d8b5b1e273b2">

**Comment by SExpert12** [comment]
Nearly 3-5 minutes if Carla is built from source.
